lib/api_repo.js

```diff
@@ -806,57 +831,85 @@ function processPaths(done, process) {
  * Finds the paths to the proto files with the given api name and version.
  *
  * If callback is set, it calls back on each of them.
- * @param {string} name the api name.
```

Comment:
Please don't remove the parameter name 😉  (this `name` is the name of the parameter -- and `the api name` is the document for the parameter).